archunit-tooling-test/src/main/java/com/tngtech/archunit/tooling/TestEngine.java

```diff
@@ -0,0 +1,12 @@
+package com.tngtech.archunit.tooling;
+
+import java.util.Set;
+
+public interface TestEngine {
```

Comment:
I was thinking, perhaps you could even implement some of the IDE test running facilities as test engines here, in case you needed to test their integration with ArchUnit. 

Stopping at build tools for now.